x-pack/plugins/cloud_security_posture/public/components/cis_benchmark_icon.tsx

```diff
@@ -30,13 +30,16 @@ const getBenchmarkIdIconType = (props: Props): string => {
     case 'cis_gcp':
       return googleCloudLogo;
     case 'cis_k8s':
-    default:
       return 'logoKubernetes';
   }
 };
 
 export const CISBenchmarkIcon = (props: Props) => (
   <EuiToolTip content={props.name}>
-    <EuiIcon type={getBenchmarkIdIconType(props)} size={props.size || 'xl'} css={props.style} />
+    <EuiIcon
+      type={getBenchmarkIdIconType(props.type) || ''}
```

Comment:
do we have a decision on how we handle missing icons and what do we want to see here in case of 3rd party providers? I think the challenge has multiple aspects. First of all `cis_*` is smth that comes from our native integration. I'm still not finished on the proposal based on wiz data, but from what I can see already, we need to start relying on another fields for the icons. Logically, it's not a benchmark icon, but rather a cloud provider icon. which also exist on Wiz, here is the list of supported cloud providers by wiz `["Alibaba", "AKS", "AWS", "Azure", "EKS", "GCP", "GKE", "Kubernetes", "Linode", "OCI", "OKE", "OpenShift", "vSphere"]` . but then the question if we want to have cloud provider icon there, or do we want Wiz icon there and just map all documents to rule.benchmark.id: wiz

Reply:
another question if we should make sure we don't render anything if type is missing , right now it looks like we will get tooltip rendered but icon with empty type

Reply:
> another question if we should make sure we don't render anything if type is missing , right now it looks like we will get tooltip rendered but icon with empty type

Fixed, thank you

Regarding the first comment, sounds like something that should be decided on by product, but it general it looks like we are going to need to provide a list of supported icons for integration creators as well, and think of how exactly we want this flyout field to be displayed when its missing. this will also have the same effect in the dashboard.

Reply:
Added this point to our sync. But anyway, we need to at least stop rendering CIS logo all the time, maybe for now based on the fact that it's not our native integration, or that benchmark_id doesn't start with `cis_*` or smth along these lines.
Long term I think we need a work item to build a component that considers [the ECS proposal](https://docs.google.com/spreadsheets/d/1p7m6c-sPn_Orgfc-jwJod9wupvF-2Qp7SDrDLle15_8/edit?gid=713700531#gid=713700531) for rule.* and `ruleset` fields